### PR TITLE
Fix Post::__get method to be compatible with Laravel 5.2.32

### PR DESCRIPTION
**Pull request by rundef**

### CODE REVIEW

**Comment by manogi** [comment]
This is looking good to me, thanks @rundef

**Comment by QWp6t** [comment]
This is actually the fix that I used locally, but I was thinking it might get confused by mutators/accessors since it's not using Eloquent's `__isset()`.

But all tests are passing so that's good. :+1:

File: src/Post.php [hunk 1]
```diff
@@ -484,7 +484,7 @@ public function getKeywordsAttribute()
      */
     public function getKeywordsStrAttribute()
     {
-        return implode(',', $this->keywords);
+        return is_null($this->keywords) ? '' : implode(',', $this->keywords);
```
Comment (QWp6t) [inline src/Post.php]: You could also do...

```php
return implode(',', (array) $this->keywords);
```

**Comment by QWp6t** [comment]
Can you tag a release so composer will grab it without having to switch off stable?

**Comment by jgrossi** [comment]
`v1.0.10` ;-)

**Comment by rundef** [comment]
Cool @jgrossi - I was about to do that :)

**Comment by jgrossi** [comment]
Sorry @rundef hehehe. You're always welcome as you already know ;-)

**Pull request merged**